### ISSUE TITLE
Narrator is announcing the wrong tooltip for the TabPage

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#5835]
.NET Core Version:
- .Net 6.0 SDK: 6.0.100-rc.1.21430.12

Have you experienced this same bug with .NET Framework?:
- No (regression from #2538)

**Repro steps:**

1. Create a Winforms .Net Core app.
2. Add a TabControl, a Tooltip, a Button to form designer.
3. Add 3 TabPages.
4. Set tooltips for pages and controls.
5. Set ShowToolTips = true.
6. Build and run app.
7. Set focus to button.
8. Hover over the TabPage.

**Actual behavior:**
The narrator announces a tooltip for TabControl
![Issue-5835](https://user-images.githubusercontent.com/23376742/134182676-f243e6d5-9178-42da-b0b0-a814ae12b7dc.gif)

**Expected behavior:**
The narrator should announces a tooltip for TabPage

**Comment by merriemcgaw** [comment]
Unless there's a customer impacted directly, I think we're OK to just put this in 7.

**Comment by Ashley-Li** [comment]
Verified with .NET SDK 7.0.100-alpha.1.22069.1 build, this issue was fixed.
![issue_verify5835 gif](https://user-images.githubusercontent.com/56664234/150490007-d0af5619-ab90-4d87-be31-cd6707aef4d6.gif)

**Comment by VinoWang01** [comment]
Verified with .NET SDK 7.0.100-preview.1.22109.10 build, this issue is fixed, the result is same as above GIF shown.
